index.js

```diff
@@ -1,11 +1,8 @@
-/*jshint node:true */
-
 'use strict';
 
 var gutil = require('gulp-util');
-var c = gutil.colors;
 var error = gutil.PluginError;
-var es = require('event-stream');
+var through = require('through2');
```

Comment:
I had users on gulp-replace mentioning that a `through2` bug might have been causing this issue https://github.com/lazd/gulp-replace/issues/24, so I'm a little wary. I don't want you to do work for no reason, however, so let's figure it out.

@shinnn, thoughts?